src/components/Reviews.jsx

```diff
@@ -1,38 +1,40 @@
 import { useEffect, useState } from "react";
 import { getReviewsList } from "../api";
+import { Routes, Route, Link, useParams } from "react-router-dom";
 
-export const Reviews = ({ loading, setLoading }) => {
+export const Reviews = ({ loading, setLoading, reviewId, setReviewId }) => {
   const [revList, setRevList] = useState([]);
 
   useEffect(() => {
-    getReviewsList().then((reviews) => {
+    getReviewsList(reviewId).then((reviews) => {
       setRevList((revs) => {
         setLoading(false);
         return reviews;
       });
     });
-  }, []);
-  console.log(revList);
+    
+  }, [reviewId]);
```

Comment:
Right now once this useEffect runs the loading state will always be false, and won't become true if we try to load another page.  To fix this at the start of the useEffect set loading to be true.

Reply:
Will do, thanks